### DISCUSSION TITLE
Cannot delete Codespaces after hitting limit

### DISCUSSION THREAD

**Discussion by sr229** [community/community#26912] [Actions]
<p>I encountered a bug whenever you open a codespace based on a PR, it returns a 404. Once I hit the limit, I couldn’t delete those phantom codespaces.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/3/39fb08a44714543ab6673b669315c04600414b9a.png" data-download-href="/uploads/short-url/8gV3ilehAUm2gZDM25KzhJBmrTs.png?dl=1" title="Screenshot 2020-09-11 214242" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/14976516/181100592-cc6fcbb8-8123-4649-882f-2d686401ee7e.png" alt="Screenshot 2020-09-11 214242" data-base62-sha1="8gV3ilehAUm2gZDM25KzhJBmrTs" width="690" height="409" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/39fb08a44714543ab6673b669315c04600414b9a_2_690x409.png, https://aws1.discourse-cdn.com/github/original/2X/3/39fb08a44714543ab6673b669315c04600414b9a.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/3/39fb08a44714543ab6673b669315c04600414b9a.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/39fb08a44714543ab6673b669315c04600414b9a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2020-09-11 214242</span><span class="informations">1007×598 53 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I opened a support ticket about it but no one responded on me for 2 days.</p>
<p>I laid out my entire experience in this <a href="https://dev.to/sr229/my-infuriating-experience-with-github-codespaces-e3a" rel="nofollow noopener">blog</a>.</p>

**Comment by matthewisabel** [comment]
<p>Hi! Sorry you’re hitting this. We’re looking into this and it appears to impacted a few folks. We’re hoping to get a fix out ASAP, I’ll circle back as we have more information.</p>

**Comment by matthewisabel** [comment, marked as answer]
<p>Update: We had some additional changes roll out that you should unblock codespace creation. If you’re still hitting issues please let me know!</p>

**Comment by sr229** [comment]
<p>Thank you! You can close my Support ticket now since I can confirm codespace creation is now working.</p>
